### ISSUE TITLE
Windows 10 Scaling causes window to not properly capture

### ISSUE THREAD

**Issue by NateEkat** [Tudi/NetworkImageProcessing#2]
If windows has the display scaling set to more that 100%, then the screen capture will not capture the whole screen. Even if in the configuration file you set it to the proper screen resolution.

**Comment by Tudi** [comment]
Hmm, never used the display scaling feature. I'm shocked anyone even tried this project. You seriously intend to use ? I'm very busy at the moment, would delay any changes if possible

**Comment by NateEkat** [comment]
Yeah, my Surface Book has a "Retina" (almost 4k, but not quite) screen, so windows by default sets the Windows scaling to 200% so text is actually readable.

I do seriously intend to use. it's been working great so far!

And that's fine, it's alright. There's another issue that I might open as well.

**Comment by Tudi** [comment]
hmm, made a test and on my laptop the capture went fine. I can only set it to 175%, but a window capture was working ok.
Are you using "desktop" capture ? Did you try to resize the input ? Any chance to share the config ? Maybe a screenshots how it looks the output on your side ? Thanks.
